### ISSUE TITLE
no nearby devices

### ISSUE THREAD

**Issue by danielellm** [Ryan-Adams365/stay-away-from-me#30]
first i love the idea of this app.

but i try the app with flutter run in debug mode on different android divices with bluetooth on.
but there find no nearby devices 

i also try it with two android phones with the same app, also no nearby devices
the phone is also vibrate all the time..

in flutter console i can see in debug mode there find bluetooth devices but in the app shows nothing..
you know about this or can help me to find a solution?

i use the newest flutter 2.05

thanks

**Comment by Ryan-Adams365** [comment]
Thanks, it was a fun hackathon project!

Unfortunately, I don't own any android devices to actually test out the bluetooth detection. When my group was building the project, only one member had access to android, so he was in charge of testing it out for that platform haha. And obviously android emulators wouldn't be able to test out the bluetooth detection functionality.

That said, when I get a chance I can take a look at the code and try to figure out what might be going on. I haven't touched the project since the hackathon, but I really liked the concept and wouldn't mind jumping back in.

**Comment by danielellm** [comment]
Yeah could be nice to Help me with this Problem.. i dont use Emulator, i
have use Tablet and Smartphone..

To be clear its detect only divices there are using the same App or every
Bluetooth ?

Ryan-Adams365 ***@***.***> schrieb am Do., 29. Apr. 2021,
18:16:

**Comment by samadurm** [comment]
It should detect every bluetooth signal, which includes any other devices which have bluetooth turned on. Did you make sure permission settings for the app to access bluetooth are on? 

As @Ryan-Adams365 mentioned, we were only able to test the Android functionality through the emulator, but I would imagine that it has something to do with the permissions settings.

Also the vibration should occur when your within 6 feet of another device.

**Comment by danielellm** [comment]
Hey thanks for your awnser..

What you mean with permission settings for the app to access bluetooth are
on?
In my Phone the App has all permission also access to Bluetooth.

I dont know is it a Flutter Version, Plugin version problem or a general
problem with Android devices.

I Check Booth, with new Plugin version and you original Version. No nearby
divices.. in the console log i can see the App is finding some around me,
but dosent Show anything in the app.





Micah Samaduroff ***@***.***> schrieb am Fr., 30. Apr. 2021,
04:05:

**Comment by creative-muhd** [comment]
@Ryan-Adams365 Hi I have a great idea that I would like to discuss with you 
Please you can email me at unofficial.muhd@gmail.com Thank you

**Comment by danielellm** [comment]
Hey, let me know

Muhammad Abu Bakr ***@***.***> schrieb am Di., 31. Mai 2022,
14:21:
